docs/how-to-call-multiple-contracts.md

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_position: 12
+---
+
+# Call Multiple Contracts in a Single Tx
+
+Up to now, we have only shown how to call one smart contract in a transaction. That is, only one input of the tx spends a smart contract UTXO, and the other inputs, if any, spend Pay-to-Public-Key-Hash ([P2PKH](https://learnmeabitcoin.com/guide/p2pkh)) UTXOs, which are generally NOT regarded as smart contracts.
+
+There are cases where it is desirable to spend multiple smart contract UTXOs in different inputs of a tx.
+
+The main differences from [calling a single contract](./how-to-deploy-and-call-a-contract.md#contract-call) are:
+
+1. Set `multiCall = true` in `MethodCallOptions`
+2. Each call may only return a partial/incomplete transaction, instead of a complete transaction
+3. A partial tx has to be passed as `CallTransation` in `MethodCallOptions` in subsequent calls
```

Comment:
Update MethodCallOptions at https://scrypt.io/scrypt-ts/how-to-test-a-contract/#call-a-public-method